dspace-server-webapp/src/main/java/org/dspace/app/rest/DiscoveryRestController.java

```diff
@@ -202,18 +203,16 @@ public RepresentationModel getFacetValues(@PathVariable("name") String facetName
         dsoTypes = emptyIfNull(dsoTypes);
 
         if (log.isTraceEnabled()) {
-            log.trace("Facetting on facet " + facetName + " with scope: " + StringUtils.trimToEmpty(dsoScope)
-                          + ", dsoTypes: " + String.join(", ", dsoTypes)
-                          + ", prefix: " + StringUtils.trimToEmpty(prefix)
-                          + ", query: " + StringUtils.trimToEmpty(query)
-                          + ", filters: " + Objects.toString(searchFilters)
-                          + ", page: " + Objects.toString(page));
+            log.info("Facetting on facet {} with scope: {}, dsoTypes: {}, prefix: {}, query: {}, filters: {}, page: {}", facetName, StringUtils.trimToEmpty(dsoScope), String.join(", ", dsoTypes), StringUtils.trimToEmpty(prefix), StringUtils.trimToEmpty(query), Objects.toString(searchFilters), Objects.toString(page));
         }
 
+        log.info("Facetting on facet {} with scope: {}, dsoTypes: {}, prefix: {}, query: {}, filters: {}, page: {}", facetName, StringUtils.trimToEmpty(dsoScope), String.join(", ", dsoTypes), StringUtils.trimToEmpty(prefix), StringUtils.trimToEmpty(query), Objects.toString(searchFilters), Objects.toString(page));
```

Comment:
This `log.info` appears to be a duplicate of the one above it?  Can we remove this one?